dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/gdv/model/allgemein/anleihen/GdvAllgemeinAnleihen.kt

```diff
@@ -0,0 +1,15 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.gdv.model.allgemein.anleihen
+
+import org.dataland.datalandbackend.model.enums.commons.YesNo
+import java.math.BigDecimal
+
+/**
+ * The data-model for the Anleihen section
+ */
+data class GdvAllgemeinAnleihen(
+    val grueneSozialeUndOderNachhaltigeEmissionen: YesNo?,
+    val ausstehendeGrueneSozialeUndOderNachhaltigeEmissionen: BigDecimal?,
+    val sustainibilityLinkedDebt: YesNo?,
```

Comment:
Is this framework written with some "denglisch" entries?

Reply:
nein steht so im excel

Reply:
sustainibility linked debt/bonds scheinen ein Finanzinstrument zu sein und sind somit ein Eigenname.

Daher wurde das vermutlich für den questionnaire nicht ins Deutsche übersetzt.

Reply:
ok